projects/ngx-charts-on-fhir/src/lib/data-layer/data-layer-color.service.ts

```diff
@@ -22,22 +24,16 @@ export class DataLayerColorService {
     }
   }
 
-  setAnnotationColor(annotation: any, color?: string): void {
-    const line = annotation;
-    line.backgroundColor = color + '33'; // temporary dirty hack to set opacity. assumes color is in 6-character hex format.
-  }
-
   addTransparency(color: string | undefined): string | undefined {
-    if (typeof color === 'string' && !color.endsWith('33')) {
-      return color + '33';
-    }
-    return color;
+    const newcolor = tinycolor(color);  
+    newcolor.setAlpha(.5);
+    return newcolor.toString();
   }
 
   setColor(dataset: Dataset, color: string): void {
     const line = dataset as Dataset<'line'>;
     line.borderColor = color;
-    line.backgroundColor = color + '33'; // temporary dirty hack to set opacity. assumes color is in 6-character hex format.
+    line.backgroundColor = this.addTransparency(color); // temporary dirty hack to set opacity. assumes color is in 6-character hex format.
```

Comment:
please remove the comment because it no longer applies

Reply:
Okay, I will remove the comments.